internal/url/url.go

```diff
@@ -7,10 +7,23 @@ import (
 	"strings"
 )
 
+const (
+	urlUnescape   = true
+	noUrlUnescape = false
+)
+
 // ParseQuery parses the URL-encoded query string and returns the corresponding map.
 // It takes separators as parameter, for example: & or ; or &;
-// It returns error if the query string is malformed.
 func ParseQuery(query string, separator byte) map[string][]string {
+	return doParseQuery(query, separator, urlUnescape)
+}
+
+// Sibling of ParseQuery, but without performing URL unescape of keys and values.
```

Comment:
Let's use godoc format

```suggestion
// ParseQueryWithoutUnescape is a sibling of ParseQuery, but without performing URL unescape of keys and values.
```